### DISCUSSION TITLE
Specify backup directory in installation instruction

### DISCUSSION THREAD

**Discussion by flaczkiDomowe** [spatie/laravel-backup#1646] [Ideas]
Hello. I had to check the code and find out, how directory "Laravel" is made inside backup folder. It seems, that it depends on env('APP_NAME'). Is there any chance that you include that information to tutorial to save ppl time? https://spatie.be/docs/laravel-backup/v8/installation-and-setup

**Comment by parallels999** [comment]
Feel free to make a PR
